parquet-hadoop/src/main/java/org/apache/parquet/hadoop/Offsets.java

```diff
@@ -68,12 +68,14 @@ public static Offsets getOffsets(SeekableInputStream input, ColumnChunkMetaData
     return new Offsets(firstDataPageOffset, dictionaryPageOffset);
   }
 
-  private static long readDictionaryPageSize(SeekableInputStream in, long pos) throws IOException {
+  private static long readDictionaryPageSize(SeekableInputStream in, ColumnChunkMetaData chunk) throws IOException {
     long origPos = -1;
     try {
       origPos = in.getPos();
+      in.seek(chunk.getStartingPos());
```

Comment:
Do we assume the dictionary page is always the chunk starting address?

Reply:
It is not obvious that one have to search this statements in the [Encoding docs](https://github.com/apache/parquet-format/blob/master/Encodings.md#dictionary-encoding-plain_dictionary--2-and-rle_dictionary--8) but it is there:
> The dictionary page is written first, before the data pages of the column chunk.

Reply:
I know it is true today, but what if that assumption is broken when more and more page types are added. Can we add something in [Encoding docs](https://github.com/apache/parquet-format/blob/master/Encodings.md#dictionary-encoding-plain_dictionary--2-and-rle_dictionary--8) to not let people change that assumption?

Reply:
I agree it should be specified more clearly and maybe not only in the Encoding doc but somewhere in the "main" page but I feel it a separate topic.

Reply:
Can you create a Jira for it @gszadovszky so that we don't lose tracking of it? 

Other than that, LGTM!

Reply:
Thanks @gszadovszky